src/components/Common/Tab.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react'
+import styled from 'styled-components'
+import remcalc from 'remcalc'
+import is from 'styled-is'
+import UnstyledButton from './UnstyledButton'
+import { Subtitle } from '../Typography'
+
+const TabContainer = styled.li`
+  display: inline-block;
+
+  & + & {
```

Comment:
Comment pls because fancy

Reply:
targetting the next sibling (I'm adding a comment in the code)